From: Scott Leggett <scott@sl.id.au>
Date: Wed, 27 Apr 2016 14:06:31 +1000
Subject: Replace bashism with POSIX shell syntax in compton-trans script.

---
 bin/compton-trans | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/compton-trans b/bin/compton-trans
index 612c35d..db41cf1 100755
--- a/bin/compton-trans
+++ b/bin/compton-trans
@@ -114,7 +114,7 @@ fi
 if test x"$action" = x'reset'; then
   xwininfo -root -tree \
   | sed -n 's/^     \(0x[[:xdigit:]]*\).*/\1/p' \
-  | while IFS=$'\n' read wid; do
+  | while IFS=$(printf '\n') read wid; do
     xprop -id "$wid" -remove _NET_WM_WINDOW_OPACITY
   done
   exit 0
